examples/nffPktgen/testing/minPktLoss.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2018-2019 Intel Corporation.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+        "flag"
+        "fmt"
+        "os"
+        "os/signal"
+        "sync/atomic"
+        "time"
+
+        "github.com/intel-go/nff-go/examples/nffPktgen/generator"
+        "github.com/intel-go/nff-go/flow"
+        "github.com/intel-go/nff-go/packet"
+)
+
+// TEST PARAMETERS:
+const TOP_SPEED uint64 = 120000000 /*(pkt/s)*/
+const TGT_LOSS float64 = 0.005 /*(0.5% target packet loss)*/
+const TRAFFIC_DELAY uint = 3 /*(sec)*/
+
+type IpPort struct {
+        Index       uint16
+        packetCount uint64
+        bytesCount  uint64
+}
+
+type HandlerContext struct {
+        port *IpPort
+}
+
+func (hc HandlerContext) Copy() interface{} {
+        return HandlerContext{
+                port: hc.port,
+        }
+}
+
+func (hc HandlerContext) Delete() {
+}
+
+func main() {
+        var (
+                speed            uint64
+                genConfig, cores string
+                inPort           uint
+                outPort          uint
+                receive          bool
+        )
+        flag.Uint64Var(&speed, "speed", TOP_SPEED, "speed of fast generator, Pkts/s")
+        flag.StringVar(&genConfig, "config", "ip4.json", "specifies config for generator")
+        flag.StringVar(&cores, "cores", "", "specifies cores")
+        flag.UintVar(&outPort, "outPort", 1, "specifies output port")
+        flag.UintVar(&inPort, "inPort", 1, "specifices input port")
+        flag.BoolVar(&receive, "receive", true, "Receive packets back and print statistics")
```

Comment:
This parameter is redundant because in binary search mode generator always receives packets, so this is always true and should be removed.